remo/dashboard/static/dashboard/js/kpi.js

```diff
@@ -0,0 +1,386 @@
+                   tip.show(d);
+               })
+               .on('mouseout', function (d, i) {
+                   tip.hide(d);
+               });
+        });
+    }
+
+    function activities_graph(key, value) {
+        var x = d3.scale.linear()
+                  .range([0, width]);
+
+        var y = d3.scale.linear()
+                  .range([height, 0]);
+
+        var xAxis = d3.svg.axis()
+                      .scale(x)
+                      .tickFormat(d3.format(',.0f'))
+                      .orient('bottom');
+
+        var yAxis = d3.svg.axis()
+                      .scale(y)
+                      .orient('left');
+
+        var area = d3.svg.area()
+                     .x(function(d) { return x(d.week); })
+                     .y0(height)
+                     .y1(function(d) { return y(d.activities); });
+
+        $('#graph-activities').text('');
+        var svg = d3.select('#graph-activities').append('svg')
+                    .attr('width', width + margin.left + margin.right)
+                    .attr('height', height + margin.top + margin.bottom)
+                    .append('g')
+                    .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
+
+        var tip = d3.tip()
+                    .attr('class', 'd3-tip')
+                    .offset([35, 0])
+                    .html(function(d) {
+                        return '<strong>Activities:</strong> <span style="color:red">' + d.activities + '</span>';
+                    });
+
+        svg.call(tip);
+
+        if ((typeof(key)!='undefined') && (typeof(value)!='undefined')) {
+            var apiurl = '/api/kpi/activities/?format=json&' + key + '=' + value;
+        } else {
+            var apiurl = '/api/kpi/activities/?format=json';
+        }
+        d3.json(apiurl, function(error, data) {
+            $('#activities-total').text(data.total);
+            $('#activities-total_quarter').text(data.quarter_total);
+            $('#activities-total_week').text(data.week_total);
+            $('#activities-percentage_week').text(data.week_growth_percentage);
+            $('#activities-percentage_quarter').text(data.quarter_growth_percentage);
+
+            var weeks = data.total_per_week;
+
+            weeks.forEach(function(d) {
+                d.week = +d.week;
+                d.activities = +d.activities;
+            });
+
+            var formatPercent = d3.format('%U');
+
+            x.domain([1, d3.max(weeks, function(d) { return d.week; })]);
+            y.domain([0, d3.max(weeks, function(d) { return d.activities; })]);
+
+            svg.append('path')
+               .datum(weeks)
+               .attr('class', 'area-activities')
+               .attr('d', area);
+
+            svg.append('g')
+               .attr('class', 'x axis')
+               .attr('transform', 'translate(0,' + height + ')')
+               .call(xAxis)
+               .append("text")
```

Comment:
Please replace double quotes with single quotes. There are a couple leftovers here and there.